google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -281,7 +308,24 @@ public ConnectionWorker(
     this.traceId = traceId;
     this.waitingRequestQueue = new LinkedList<AppendRequestAndResponse>();
     this.inflightRequestQueue = new LinkedList<AppendRequestAndResponse>();
+    this.retryFirstDelay = retryFirstDelay;
```

Comment:
We don't need to store them, store a retrySettings is enough

Reply:
Done